compiler/src/lib.rs

```diff
@@ -343,6 +343,11 @@ pub fn inputs_to_query_callback<T: FieldElement>(inputs: Vec<T>) -> impl Fn(&str
                 print!("{}", items[1].parse::<u8>().unwrap() as char);
                 Some(0.into())
             }
+            "\"hint\"" => {
+                assert_eq!(items.len(), 2);
+                let hint = items[1].parse::<u64>().unwrap();
```

Comment:
`u64` might not be the right type. Can we use the parser of `T`'s underlying integer type?

Reply:
Right! Apparently there is `T::from_str()`! Still struggling to find stuff like that easily. I guess we could have a `FieldElement: FromStr` trait bound, then you could do `items[1].parse::<T>().unwrap();`?